sdk/communication/Azure.Communication.CallingServer/src/Models/CallConnectionProperties.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Communication.CallingServer
+{
+    /// <summary> The Details of the call. </summary>
+    public class CallConnectionProperties
+    {
+        /// <summary> Initializes a new instance of CallConnectionProperties. </summary>
+        /// <param name="callConnectionPropertiesInternal">The call connection properties internal.</param>
+        internal CallConnectionProperties(CallConnectionPropertiesInternal callConnectionPropertiesInternal)
+        {
+            /*
```

Comment:
Should we leave this in here commented out or remove it?

Reply:
Removed